react/__tests__/Telemarketing.test.tsx

```diff
@@ -43,7 +43,128 @@ describe('<Telemarketing /> component', () => {
         onInputChange={() => {}}
         onSetSession={() => {}}
       />
-    ).asFragment()
-    expect(component).toMatchSnapshot()
+    )
+    expect(asFragment()).toMatchSnapshot()
+  })
+
+  it('should show attendant email', () => {
+    const email = 'attendant@vtex.com'
+    const { getByText } = render(
+      <Telemarketing
+        attendantEmail={email}
+        emailInput="email@vtex.com"
+        intl={intl}
+        loading={false}
+        onDepersonify={() => {}}
+        onInputChange={() => {}}
+        onSetSession={() => {}}
+      />
+    )
+
+    const attendantEmail = getByText(email)
+    expect(attendantEmail).toBeTruthy()
```

Comment:
Since we have the e-mail here, would not be better to use [toBe](https://stackoverflow.com/questions/32615713/tobetrue-vs-tobetruthy-vs-tobetrue)?

Reply:
`getByTest` returns an `HTMLElement` not a `String` that I'm using to find it. That's why I dont use `toBe` I do not have something to compare, just know that it has the email text.